qt/scientific_interfaces/Indirect/IndirectFunctionBrowser/ConvFunctionModel.cpp

```diff
@@ -555,6 +556,21 @@ std::string ConvFunctionModel::buildTeixeiraFunctionString() const {
          "constraints=(Height>0, DiffCoeff>0, Tau>0)";
 }
 
+std::string ConvFunctionModel::buildFickFunctionString() const {
+  return "name=FickDiffusionSQE, Height=1, DiffCoeff=2.3, Centre=0, "
+         "constraints=(Height>0, DiffCoeff>0)";
+}
+
+std::string ConvFunctionModel::buildChudleyElliotString() const {
+  return "name=ChudleyElliotSQE, Height=1, Tau=1.25, Centre=0, L=1.0, "
+         "constraints=(Height>0, Tau>0)";
+}
+
+std::string ConvFunctionModel::buildHallRossString() const {
+  return "name=HallRossSQE, Height=1, Tau=1.25, Centre=0, L=1.0, "
+         "constraints=(Height>0, Tau>0)";
```

Comment:
Probably it would make sense to set constraints on "L" to be positive, as well.

Reply:
Good idea!